docs/content/docs/start/tree-start.md

```diff
@@ -0,0 +1,239 @@
+---
+title: SharedTree Quick Start
+menuPosition: 2
+codeCopyButton: true
+aliases:
+  - "/docs/get-started/tree-quick-start/"
+  - "/start/tree-quick-start/"
+  - "/docs/tree-start/"
+
+---
+
+`SharedTree` is a [distributed data structure](../data-structures/overview.md) that looks and feels like simple JavaScript objects with a type safe wrapper.
+This guide will walk you through the basics of creating, configuring, and interacting with a `SharedTree` in your application. 
+
+## Creating a Tree
+
+A `SharedTree` can be created by defining a `ContainerSchema` with an initial object of type `SharedTree` and using this schema to create and load your container.
+This example creates a container using an Azure specific client.
+
+See more info on creating and loading containers [here](../build/containers.md#creating-a-container).
+
+```
+import { AzureClient } from "@fluidframework/azure-client";
+import { ContainerSchema, SharedTree } from "fluid-framework";
+
+// clientProps specifies connection details and is defined by the app author
+const client = new AzureClient(clientProps);
+
+const containerSchema: ContainerSchema = {
+	initialObjects: {
+		appData: SharedTree,
+	},
+};
+
+const { container } = await client.createContainer(containerSchema, "2");
+```
+
+After creating a `SharedTree`, you need to create a `TreeView`.
+A `TreeView` provides the interface for reading and editing data on the `SharedTree` using a particular schema.
+This is done by calling `viewWith` on the `SharedTree` with your tree configuration.
+
+```
+const treeConfiguration = new TreeViewConfiguration(
+	{ schema: RootSchema }
+)
+
+const appData = container.initialObjects.appData.viewWith(
+	treeConfiguration
+);
+```
+
+The tree configuration takes in a schema for the root of the tree which will need to be defined by your application.
+
+## Defining a Schema
+
+A schema outlines the structure and types of data that your tree will manage.
+Having a schema provides useful guarantees about the shape your data will take.
+It is also used to generate TypeScript types which means the structure of your data is analogous to JS data types for easy use in external libraries or systems.
+
+To define a schema, first create a `SchemaFactory` with a unique string to use as the namespace.
+
+```
+const schemaFactory = new SchemaFactory("some-schema-id-prob-a-uuid")
+```
+
+`SchemaFactory` provides some methods for specifying internal nodes including `object()`, `array()`, and `map()`; and five primitive data types for specifying leaf nodes: `boolean`, `string`, `number`, `null`, and `handle`.
+See [schema definition](../data-structures/tree#schema-definition) for more info on the provided types.
+
+You can define a schema in customizable mode by extending the notional type object which also creates a TypeScript data type.
```

Comment:
Mentioning customizable mode in passing without defining it is a bit confusing. I would probably not mention that this is a named mode and instead (at most) mention POJO mode as a brief aside.

Reply:
```suggestion
You can define a schema in customizable mode by extending one of the built-in object types.
When you specify the structure of your schema type this way, the system also creates a TypeScript type for type-safe usage in your application.
```